ui/raidboss/data/07-dt/raid/r5s.txt

```diff
@@ -1,9 +1,364 @@
-### Arcadion (R5S): AAC Cruiserweight M1 Savage
+### AAC CRUISERWEIGHT M1 (SAVAGE)
+# ZoneId: 1257
+
+# -ii 93C1 93C2 93C3 93C8 93C9 A327 A70A A70B A70C A70D A77A
+
+# The X-snap twist syncs are ugly, sorry
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
 
+# Intro
 0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+15.3 "Deep Cut" Ability { id: "A722", source: "Dancing Green" }
+20.9 "--middle--" Ability { id: "A6C5", source: "Dancing Green" }
+27.0 "Flip to A-side/Flip to B-side" Ability { id: ["A780", "A781"], source: "Dancing Green" }
+34.1 "Snaps" Ability { id: ["A728", "A729", "A72A", "A4DB", "A72B", "A72C", "A72D", "A4DC", "A730", "A731", "A732", "A4DD", "A733", "A734", "A735", "A4DE", "A739", "A73A", "A73B", "A4DF", "A73C", "A73D", "A73E", "A4E0"], source: "Dancing Green" } duration 2
+37.8 "Twist" #Ability { id: "A738", source: "Dancing Green" }
```

Comment:
Shouldn't this be the `Drop the Needle` part?

Reply:
All 3/4/5 hits of the snap and twist mechanic have the same name. The original output for the enrage pull I did the first pass against was:

```
26.8 "Flip to B-side" Ability { id: "A781", source: "Dancing Green" }
33.9 "2-snap Twist & Drop the Needle" Ability { id: "A72A", source: "Dancing Green" }
35.5 "2-snap Twist & Drop the Needle" Ability { id: "A72E", source: "Dancing Green" }
37.5 "2-snap Twist & Drop the Needle" Ability { id: "A72F", source: "Dancing Green" }
39.3 "Play B-side" Ability { id: "A784", source: "Dancing Green" }
```

So I'm fine with calling this "Drop the Needle" in the timeline, but there's no way to leave the original name of the ability in-place and do a timelineReplace to adjust it.

Reply:
I think we have the same idea here?  I just meant that to me, I think of the final hit as the `Drop the Needle` part and less the `Twist` part.  I guess it's kind of both, maybe the timeline should be `2/3/4-snap` with `duration 2` and the final hit should be the `Twist & Drop the Needle`?

Reply:
I'm fine with what I've adjusted to in latest commits if that makes sense to you?

Reply:
Yeah that looks fine.